packages/flutter/test/painting/image_stream_test.dart

```diff
@@ -128,7 +139,54 @@ void main() {
     expect(mockCodec.numFramesAsked, 1);
   });
 
-  testWidgets('Chunk events are delivered', (WidgetTester tester) async {
+  testWidgets('Chunk events of base ImageStreamCompleter are delivered', (WidgetTester tester) async {
+    final List<ImageChunkEvent> chunkEvents = <ImageChunkEvent>[];
+    final StreamController<ImageChunkEvent> streamController = StreamController<ImageChunkEvent>();
+    final ImageStreamCompleter imageStream = FakeEventReportingImageStreamCompleter(
+      chunkEvents: streamController.stream,
+    );
+
+    imageStream.addListener(ImageStreamListener(
+          (ImageInfo image, bool synchronousCall) { },
```

Comment:
nit formatting: put this on the same indentation as the next line.

Reply:
@goderbauer There is no formatting tool for Flutter right?